modules/collabora_online_group/tests/src/Traits/GroupRelationTrait.php

```diff
@@ -43,4 +43,14 @@ protected function createPluginRelation(GroupTypeInterface $group_type, string $
     return $entity;
   }
 
+  /**
+   * Checks if 'group_relationship_type' group type entity exists.
+   *
+   * @return bool
+   * If the entity type exists.
+   */
+  protected function isGroupRelationshipType(): bool {
```

Comment:
To me, the method name is confusing.
And the two usages are not actually the same.

In one case we really want to know if the entity type 'group_relationship_type' exists. So it is primarily feature detection, and it incidentally maps to versions of the group module.

In the other case we use the feature detection as a proxy for group module version detection, and we are actually interested in another change, a widget label somewhere, which may be a function of the group module version.

An alternative name could be `groupRelationshipTypeEntityTypeExists()`, but I also don't like it very much.

I think we should just keep the inline code in the first instance, and then for the second instance we simply allow both labels.

Reply:
Dropping the function, is a bit confusing, I allow both labels, only side effect is that we are not asserting exact values which means that of the values could be removed and we wouldn't know, but is not the goal of the test.